teeio-validator/teeio_validator/test_config/test_config_pcrc.c

```diff
@@ -84,24 +83,18 @@ bool test_config_pcrc_disable_sel(void *test_context)
 
 bool test_config_pcrc_support_sel(void *test_context)
 {
-  return test_config_support_common(test_context) &&
-          test_config_check_pcrc_support_common(test_context);
+  return test_config_check_pcrc_support_common(test_context);
 }
 
 bool test_config_pcrc_check_sel(void *test_context)
 {
-  return test_config_check_common(test_context, "PCRC Config Assertion");
+  return true;
```

Comment:
How PCRC is related to config combination?

Since it is a different issue, please split PCRC change to a different PR.